test/com/dataengi/crm/identities/daos/UsersDAOSpec.scala

```diff
@@ -17,42 +17,141 @@ class UsersDAOSpec
     with CompaniesServiceContext
     with RolesServiceContext {
 
-  val TestLoginInfo: LoginInfo = LoginInfo("provider", "dasd@asd.com")
-  lazy val usersDAO            = application.injector.instanceOf[UsersDAO]
-
   "UsersDAOSpecification" should {
 
     "add user" in {
-      val NewCompanyName: String          = "NEW_COMPANY" + Gen.alphaStr.sample.get
-      val newCompanyResult: XorType[Long] = companiesService.create(NewCompanyName).await()
-      val newCompany                      = companiesService.get(newCompanyResult.value).await().value
-      val role                            = rolesService.find(RolesConfiguration.CompanyManager.name).await().value
+      val fixture = new Fixture(application, userCount = 1)
+      import fixture._
```

Comment:
Why didn't you use `Scope`?
https://etorreborre.github.io/specs2/guide/SPECS2-4.0.2/org.specs2.guide.Scopes.html

Reply:
Good idea